src/format/rulevalues.d

```diff
@@ -66,10 +66,7 @@ public:
 	}
 
 	@property
-	size_t frozen(size_t f) in {
-		assert(f > 0 && f <= capacity);
-		assert(this[f - 1]);
-	} do {
+	size_t frozen(size_t f) in(f > 0 && f <= capacity) in(this[f - 1]) {
```

Comment:
Maybe they can all be merged in one contract? As in `f > 0 && f <= capacity && this[f - 1]`?

Reply:
Is this really what we want here? I feel like this merges two only so much related checks into one (but I could be wrong).

Reply:
Yeah, idk. Several `in` in line, just isn't very readable.

Reply:
Maybe we could keep this one in oldskool contract syntax then? (until a potential overhaul of the formatter in the future)

Reply:
Yes I think keeping the one that don't look good with the old style is the way to go here.

Reply:
reverted